stacker/providers/aws/default.py

```diff
@@ -72,6 +72,54 @@ def _throttling_checker(exc):
                               retry_checker=_throttling_checker)
 
 
+def s3_fallback(self, fqn, template_url, parameters, tags, create_or_update, **kwargs):
```

Comment:
You could replace `create_or_update` with `method` and pass the actual method that will be used in.  They have the same basic arguments, no matter which you call, except for the create_changeset method, which requires an extra argument (which you already pass in via kwargs).